src/main/java/chat/Chat.java

```diff
@@ -0,0 +1,64 @@
+package chat;
+
+import chat.command.Command;
+import chat.TaskList;
+import chat.Storage;
+import chat.Ui;
+
+/**
+ * Main class that executes Chat the Cat.
+ */
+
+public class Chat {
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Initialise Chat the Cat by loading data from file to create list of tasks.
+     * 
+     * @param filePath Path of the file that is read from.
+     */
+    public Chat(String filePath) {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        try {
+            tasks = new TaskList(storage.load());
+        } catch (ChatException e) {
+            ui.showLoadingError();
+            tasks = new TaskList();
+        }
+    }
+
+    /**
+     * Method that starts and runs Chat the Cat.
+     */
+    public void run() {
+        ui.lines();
```

Comment:
Could consider calling lines() in the greet() method (and other places where you call it) instead of in the main, where lines are very precious. ie. in the greet method:
`greet() {
ui.lines();
...
ui.lines()
}`